test/onnx/test_op_consistancy.py

```diff
@@ -398,12 +398,21 @@ def test_output_match(self, device: str, dtype: torch.dtype, op):
             requires_grad=False,
         )
 
-        for i, (opset, cpu_sample) in enumerate(
-            itertools.product(TESTED_OPSETS, samples)
-        ):
+        if dtype == torch.bfloat16 and opset < 13:
+            # Always skip bfloat16 for opsets before 13 because onnx started
+            # supporting bfloat16 from opset 13.
+            self.skipTest("bfloat16 not supported before opset 13")
+
+        context_manager = contextlib.nullcontext()
```

Comment:
And I couldn't find a good way to eliminate the context_manager, since common_device_type.ops still cannot handle opsets.

Reply:
We could potentially duplicate test_output_match programmatically and patch versions that are tied to different opsets back to the class. That requires

1. Duplicate ops_db for each opset version
2. programmatically take test_output_match and pre-apply opset, as well as modifying skip_ops to modify each ops_db based on the opset
3. call common_device_type.ops with different `ops_db`s